packages/server/lib/studio.ts

```diff
@@ -71,3 +79,10 @@ export const save = (saveInfo: SaveInfo) => {
     }
   })
 }
+
+export const getStudioDetails = async (fileDetails: FileDetails) => {
+  return {
+    studioExtended: await wasTestExtended(fileDetails).catch(() => false),
+    studioCreated: await wasTestCreated(fileDetails).catch(() => false),
+  }
+}
```

Comment:
This is called for each test? 
and do both `wasTestExtended` and `wasTestCreated` read the file or do they sharing the read?
So for a spec file of 60 tests, is this going to read the file from disk 120 times?
the performance test in this PR ensures that a single test takes less than 100ms to parse the ast from an already in memory file, but I could see this being an issue